.github/workflows/report_tasks.yml

```diff
@@ -0,0 +1,49 @@
+# Run various tasks related to the report aggregations
+
+name: SQL tasks for report
+on:
+  workflow_dispatch:
+    inputs:
+      Task:
+        type: choice
+        description: "The SQL task to perform"
+        required: true
+        options:
+          - "hello_world"
+          - "bad_sql"
+          - "danger"
+          - "danger2"
+      Environment:
+        type: choice
+        description: "The environment to target"
+        required: true
+        default: 'qa'
+        options:
+          - 'qa'
```

Comment:
Is this is for the main H DB there's no QA env there.

Reply:
Yeah, there's no QA, but it means we can deploy the code as far as QA without releasing it fully and see what happens. This won't look like this in the end I don't think.